src/components/Picker/index.js

```diff
@@ -14,22 +15,40 @@ const propTypes = {
     /** Should the picker appear disabled? */
     isDisabled: PropTypes.bool,
 
-    /** Should the input be styled for errors  */
-    hasError: PropTypes.bool,
+    /** Input value */
+    value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
Value will be `undefined` when using it with Form.

Reply:
`undefined` is implicitly specified  when `isRequired` is not set. Do you want me to add it explicitly?

Reply:
yes please, add it explicitly